kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
             'kube_pod_container_resource_requests_memory_bytes': 'container.memory_requested',
             'kube_pod_container_status_ready': 'container.ready',
             'kube_pod_container_status_restarts': 'container.restarts',
+            'kube_pod_container_status_restarts_total': 'container.restarts',
```

Comment:
For future reference, can you add `// 1.2+` to this line, and `// up to 1.1` to the previous, so we document it's intentional?